arduino_code/led_dimmer/led_dimmer.ino

```diff
@@ -37,6 +46,31 @@
 
 char i2c_last_received = I2C_IDDLE;
 
+
+/* board IDs
+0: all off
+1: all off + wake up slave
+2: all on
+3: all on + wake up slave
+4: wake up master
+*/
+
+//board id jumper settings
+int jumpers[]={ADDR1, ADDR2, ADDR3, ADDR4, ADDR5, ADDR6, ADDR7};
+int board_id = 0;
+
+for (int x : jumpers){
+    int jumper_value = !digitalRead(x);
+    board_id +=1
+     if (jumper_value == HIGH){ 
+        break;
+     }
```

Comment:
Dat zou ik anders doen. Ik zou iedere (groep) jumper positie een betekenis geven.
bijvoorbeeld: de eerste twee jumpers geven samen vier codepoints:
```
00: all off
01: all on
10: wake up light
11: reserved
```
De derde jumper geeft dan twee codepoints:
```
0: wake up slave disabled
1: wake up slave enabled
```
Op die manier gebruik je minder jumpers, en is het (voor mij) ook makkelijker te begrijpen.
Dit werkt natuurlijk ook, en je kan het gewoon zo laten.

Ter info: de encoding die je hier gebruikt noemt ["one-hot"](https://en.wikipedia.org/wiki/One-hot)